src/rpc/blockchain.cpp

```diff
@@ -373,7 +373,7 @@ std::string EntryDescriptionString()
            "        \"base\" : n,         (numeric) transaction fee in " + CURRENCY_UNIT + "\n"
            "        \"modified\" : n,     (numeric) transaction fee with fee deltas used for mining priority in " + CURRENCY_UNIT + "\n"
            "        \"ancestor\" : n,     (numeric) modified fees (see above) of in-mempool ancestors (including this one) in " + CURRENCY_UNIT + "\n"
-           "        \"descendent\" : n,   (numeric) number of in-mempool ancestor transactions (including this one) in " + CURRENCY_UNIT + "\n"
+           "        \"descendent\" : n,   (numeric) number of in-mempool descendent transactions (including this one) in " + CURRENCY_UNIT + "\n"
```

Comment:
`GetModFeesWithDescendants` has nothing to do with the number of transactions, it returns a fee.